dspace-iiif/src/main/java/org/dspace/app/iiif/service/AbstractResourceService.java

```diff
@@ -43,8 +43,8 @@
     protected static final String THUMBNAIL_PATH = "/full/90,/0/default.jpg";
     protected static final String IMAGE_PATH =     "/full/full/0/default.jpg";
     // Default canvas dimensions.
-    protected static final Integer DEFAULT_CANVAS_WIDTH_FALLBACK = 1200;
-    protected static final Integer DEFAULT_CANVAS_HEIGHT_FALLBACK = 1600;
+    protected static Integer defaultCanvasWidthFallback = 2200;
```

Comment:
It's true that the portrait aspect ratio is more common since books and documents have lots of images. What I did was test to see if a portrait or a landscape ratio (like this one) got the best result overall. It seemed like a landscape ratio (roughly similar to the aspect ratio of the Mirador viewer window) gave better results. But I wouldn't place too much weight on that observation.